package.json

```diff
@@ -19,11 +19,15 @@
     "@astrojs/db": "^0.11.6",
     "@astrojs/netlify": "^5.3.2",
     "astro": "^4.11.0",
+    "crypto": "^1.0.1",
```

Comment:
Crypto is now a built-in nodejs library. 

crypto and crypto-js are both discontinued libraries.